src/main/kotlin/racingcar/RacingCarGame.kt

```diff
@@ -0,0 +1,26 @@
+package racingcar
+
+import racingcar.domain.Race
+import racingcar.domain.RaceSetting
+import racingcar.view.InputView
+import racingcar.view.ResultView
+
+class RacingCarGame {
+    companion object {
+        @JvmStatic
+        fun main(args: Array<String>) {
+            val racingCarGameInput = InputView.getInput()
+
+            val numberOfCarInput = racingCarGameInput.numberOfCar
+            val totalCountOfRoundInput = racingCarGameInput.totalCountOfRound
```

Comment:
두 입력을 굳이 하나의 데이터 클래스로 받을 필요가 없다고 생각해요. 🤔

```suggestion
            val numberOfCarInput = InputView.InputNumberOfCar()
            val totalCountOfRoundInput = InputView.inputTotalCountOfRound()
```